sdk/core/azure-core/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 ### Features Added
 
+- Added support for HTTP proxy servers, both unauthenticated and with Plain authentication.
+- Added universal support for several TLS options:
+  - Added the ability to set the expected TLS root certificate for TLS connection (useful if a proxy server uses a TLS certificate that is not chained to a known root).
+  - Added the ability to enable TLS certificate revocation list checks (off by default).
```

Comment:
Are all these features directly user-facing? If not - let's list the ones that are, move the rest to "Other" section, maybe summarizing them on the top as something like "Added initial foundation for WebSocket support".

Reply:
Yes, these are all user facing. These are changes to either the `TransportOptions` or `CurlTransportOptions` types, both of which are customer facing.

Very little of the functionality in this pull request is WebSockets related functionality (the only exception is removing "final" from CurlTransport, and WinHttpTransport and a couple of added methods.

Almost everything here is functionality that Cognitive asked for, but functionality (like proxy server support) is core functionality.